src/applications/static-pages/sass/static-pages.scss

```diff
@@ -77,4 +77,13 @@ $formation-image-path: '~@department-of-veterans-affairs/formation/assets/img';
   > span {
     display: none;
   }
+
+  &.form-expanding-group-open.borderless-alert {
```

Comment:
I'm the `borderless-alert` class is getting added to the relevant `form-expanding-group` elements in department-of-veterans-affairs/vagov-content#378

Reply:
good call, I'll check 👍

Reply:
good call, I'll check 👍